src/app.config.ts

```diff
@@ -1,17 +1,19 @@
-import { configFactory, toNumber, toPem, toString } from './config'
+import { configFactory, toArray, toNumber, toPem, toString } from './config'
 
 const appConfig = {
   apiToken: toString(),
   sslKey: toPem(),
   sslCert: toPem(),
   httpPort: toNumber(),
   httpsPort: toNumber(),
+  gsuiteScopes: toArray(),
```

Comment:
This should not be part of the app config.

Reply:
you are right, i've added a separate config provider for diag endpoint instead.